src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlManagedInstanceKeyVaultKey.md

```diff
@@ -0,0 +1,146 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/Add-AzureRmSqlManagedInstanceKeyVaultKey
+schema: 2.0.0
+---
+
+# Add-AzureRmSqlManagedInstanceKeyVaultKey
+
+## SYNOPSIS
+Adds a key vault key to the provided Managed Instance. 
+## SYNTAX
+
+```
+Add-AzureRmSqlManagedInstanceKeyVaultKey [-ResourceGroupName] <String> [-ManagedInstanceName] <String>
+ [-KeyId] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Add-AzureRmSqlManagedInstanceKeyVaultKey cmdlet adds a key vault key to the provided Managed Instance. The managed instance must have 'get, wrapKey, unwrapKey' permissions to the vault.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Add-AzureRmSqlManagedInstanceKeyVaultKey -ResourceGroupName 'ContosoResourceGroup' -ManagedInstanceName 'ContosoManagedInstanceName' -KeyId 'https://contoso.vault.azure.net/keys/contosokey/01234567890123456789012345678901'
```

Comment:
Please add the call to address this requirement "The managed instance must have 'get, wrapKey, unwrapKey' permissions to the vault"

Reply:
Ping